tests/test-libs.sh

```diff
@@ -104,3 +104,8 @@ call_base_test()
 
     . $(dirname $(readlink -e ${BASH_SOURCE[2]}))/$fname
 }
+
+allow_taint()
+{
+    touch /tmp/allow_taint
```

Comment:
This will run on the host as well, outside the vm, and pollute /tmp there.

I'd do it differently, have it be a parameter that's returned by the 'deps' subcommand; that way it doesn't need the filesystem at all.